packages/lib-classifier/src/plugins/tasks/DrawingTask/models/drawingTools/Point.js

```diff
@@ -1,12 +1,29 @@
 import { types } from 'mobx-state-tree'
+import { Point as PointMark } from '../markings'
 
 const Point = types.model('Point', {
   color: types.optional(types.string, ''),
   label: types.optional(types.string, ''),
+  marks: types.map(PointMark),
   max: types.maybe(types.union(types.string, types.number), ''),
   min: types.maybe(types.union(types.string, types.number), ''),
   size: types.maybe(types.enumeration(['large', 'small'])),
   type: types.literal('point')
```

Comment:
There's a bunch of tool properties that are shared between the point and line tools, which could be moved into a common Tool model.

Reply:
Redoing the drawing tool models probably rates its own PR based off either this or #1308.